Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "NEOs"
 uuid = "b41c07a2-2abb-11e9-070a-c3c1b239e7df"
 authors = ["Jorge A. Pérez Hernández", "Luis Benet"]
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
I suggest to leave this as "0.3"; in includes all 0.3.x versions.

Reply:
Done.